jjb/jobs/qcs-nightly-automation.yaml

```diff
@@ -0,0 +1,57 @@
+- job:
+    name: 'qcs-nightly-automation'
+    node: 'f25-np'
+    triggers:
+        - timed: '@midnight'
+        - pollscm:
+            cron: 'H/30 * * * *'
```

Comment:
I think 30 minutes is too short to changes happen. I think we can either leave only to run `@midnight` (the one I like most) or leave both but increase the SCM polling to maybe every one or two hours.